app/services/winning_vendor_update_auction.rb

```diff
@@ -11,10 +11,12 @@ def perform
     if user_is_winning_bidder? && status.present?
       auction.update(delivery_status: :pending_acceptance)
     elsif user_is_winning_bidder? && delivery_url.present?
-      auction.update(
+      update = auction.update(
         delivery_url: delivery_url,
         delivery_status: :work_in_progress
       )
+      AdminMailer.vendor_started_work(auction: auction).deliver_later
+      update
```

Comment:
are we returning this bc we want it to return `true`? if so, what about returning `true` here instead?